src/scripts/components/autoplay/autoplay.tsx

```diff
@@ -1,17 +1,17 @@
 import {FunctionComponent} from 'react';
-import {useInterval} from '../../hooks/use-interval';
 import {useHistory} from 'react-router-dom';
 
-interface Props {
-  autoPlayLink: string | null;
-}
+import {useInterval} from '../../hooks/use-interval';
+import {useStoryNavigation} from '../../hooks/use-story-navigation';
```

Comment:
i would prefer to pass the autoPlayLink as it was before because the parent StoryPagination component already calls the useStoryNavigation hook. This way the Autoplay component can stay "dumb" which is usually a good thing.